pyproject.toml

```diff
@@ -89,13 +89,12 @@ dependencies = [
     "kubernetes>=30.1.0",
     "firecrawl-py>=0.0.16",
     "json-repair>=0.25.2",
-    "langchain-nvidia-ai-endpoints==0.1.6",
+    "langchain-nvidia-ai-endpoints>=0.1.6",
```

Comment:
I'd say find the version that works with this update and lock it on that.
Same for the others

Reply:
Any reason you recommend locking to a specific version? I ask because we ran into this several times with the previous project, where users doing dev work pulled in specific upgraded versions of dependencies. I suspect that would happen much less often with Langflow, but curious why we wouldn't keep them as relaxed as possible